docs/modules/deployment/pages/minion/secure-karaf.adoc

```diff
@@ -10,6 +10,7 @@ ssh -p 8201 admin@localhost
 config:edit org.apache.karaf.jaas
```

Comment:
The addition looks good, but I have comments on the rest of the file (since the text is short).

Line 1: Login should be "Log in"
Line 24: add a period to the end of the sentence.
Line 25: Line should be "Log out and log back in to verify that your new password is set."

Normally we put periods at the end of all sentences, but I notice that the rest of the deployment guide doesn't do this for this type of procedure, so we can leave them off for now to be consistent.

Reply:
done